roxie/ccd/ccdcontext.cpp

```diff
@@ -2147,6 +2147,18 @@ class CRoxieContextBase : implements IRoxieAgentContext, implements ICodeContext
         }
         return timer;
     }
+    virtual ISectionTimer * registerStatsTimer(unsigned activityId, const char * name, unsigned int statsOption)
```

Comment:
again (and in all other non-trivial occasions) worth re-implementing the old function by calling the new.